MobileCenter/MobileCenter/Internals/Channel/MSChannel.h

```diff
@@ -34,8 +34,8 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @return A new `MSChannelDefault` instance.
  */
-- (instancetype)initWithSender:(id<MSSender>)sender
-                       storage:(id<MSStorage>)storage
+- (instancetype)initWithSender:(nullable id<MSSender>)sender
```

Comment:
For those two I think I would rather have nonnull sender and storage here and in the logmanager properties. I don't think of one use case where those can be null.